dateparser/date_parser.py

```diff
@@ -2,14 +2,15 @@
 from __future__ import unicode_literals
 
 import re
-
 from datetime import datetime
+from six import binary_type
```

Comment:
Can we please use `binary_type = bytes if sys.version_info[0] == 3 else str  # PY3 compatibility` here to not introduce new dependency? We would address compatibility issue in https://github.com/scrapinghub/dateparser/pull/15 some time later.